hips/tiles/tile.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from .description import HipsDescription
+import numpy as np
+
+__all__ = [
+    'HipsTile',
+]
+
+
+class HipsTile:
+    """HiPS tile container.
+
+    """
+
+    def __init__(self, hips_description: HipsDescription, order: int, ipix: int, format: str) -> None:
+        self.hips_description = hips_description
+        self.order = order
+        self.ipix = ipix
+        self.format = format
+
+    @property
+    def base_url(self) -> str:
+        """HiPS tile base url (`str`)."""
+        return self.base_url
+
+    @property
+    def data(self) -> list:
+        """HiPS tile data (`list`)."""
+        return self.data
+
+    @property
+    def tile_url(self) -> str:
+        """HiPS tile url (`str`)."""
+        directory = np.around(self.ipix, decimals=-(len(str(self.ipix)) - 1))
```

Comment:
Split out this line computing `directory` into a separate "private" method:
```
@staticmethod
def _directory(ipix):
    """Directory of the tile (int)"""
    return np.around(ipix, decimals=-(len(str(ipix)) - 1))
```
The advantage is that then it's easier to test and (if needed, possibly) can be called from other places than `tile_url` (e.g. from a `tile_filename` you'll need later to access tiles from local disk).